dev/com.ibm.ws.webcontainer/test/com/ibm/ws/webcontainer/webapp/SRTServletContainerVersionTest.java

```diff
@@ -24,32 +26,51 @@
 import com.ibm.ws.webcontainer.osgi.WebContainer;
 
 /**
- *
+ * OLGH PR 20004 -- Removed ServiceReference
  */
 public class SRTServletContainerVersionTest {
 
     private final Mockery context = new Mockery();
     private int mockId = 1;
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testServlet50ObjectCreation() throws Exception {
+        WebContainer webContainer = new WebContainer();
+
+        Class<WebContainer> clazz = (Class<WebContainer>) webContainer.getClass();
+
+        Field versionSetter = clazz.getDeclaredField("loadedContainerSpecLevel");
+
+        versionSetter.setAccessible(true);
+        versionSetter.set(webContainer, 50);
```

Comment:
This is a static field so you don't have to pass an object to the Field.set method.  I think it is more clear to pass null here to make that explicit.

Reply:
Ah, you're right. Updating